Automattic/wp-calypso
# Jetpack Cloud: Add Monitor product lightbox, product data

***UPDATE:** D117122-code is deployed. All relevant instructions have been struck through.*

~~**NOTE: This PR is not fit to merge until a product has been created for Jetpack Monitor in the WordPress.com Store.**~~

~~Depends on D117122-code.~~
Resolves `1204992567518369-as-1204995327932145`.

## Proposed Changes

* Add a section for disclaimer text to `JetpackProductInfo`.
* Add a property `alwaysExpanded` to `JetpackProductInfoSection`, to force a section to always render in its expanded state.
* Add product and feature slugs for Jetpack Monitor.
* Add lightbox verbiage and icons for Jetpack Monitor.

## Testing Instructions

~~***NOTE:** To test before D117122-code is merged, you'll need a WordPress.com sandbox. For prerequisite test instructions, consult the aforementioned PR.*~~

1. In your test environment, visit the Partner Portal's **Issue License** page (i.e., `/partner-portal/issue-license`).
2. Verify you see Monitor in the list of issuable products.
3. Click the "More about Monitor" link to open its lightbox.
4. Verify the lightbox opens, and that all its verbiage and icons are rendered the same as in its design (Gk3oZYJHerhUZ2KVabMAEx-fi-4016-92602).
5. Click the **Select License** button; verify the lightbox closes and the product card's checkbox is now checked.

### Reference screenshots

<img height="370" src="https://github.com/Automattic/wp-calypso/assets/670067/73f0842e-136a-4e2a-ab23-a71d6447bf9e" />

<img height="370" src="https://github.com/Automattic/wp-calypso/assets/670067/c3829e68-612a-45bf-b167-8f0c1049bd14" />

<img height="370" src="https://github.com/Automattic/wp-calypso/assets/670067/2e9b148c-269c-4886-869d-100170da7bc9" />

<img height="370" src="https://github.com/Automattic/wp-calypso/assets/670067/fe11435b-0a71-437c-bda8-da9fd04aa141" />
